domain/src/main/java/com/example/domain/repository/IHeartRateMeasureRepository.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.domain.repository
+
+import com.example.domain.model.HeartRateInfo
+import  com.example.domain.model.Result
+interface IHeartRateMeasureRepository {
+    suspend fun getHeartRateMeasureData(name:String):Result<HeartRateInfo>
+}
```

Comment:
Where is no saving function for heart rate?

Reply:
That will cover i upcoming PR as i am working on room operation in another branch

Reply:
Ok but for the future we need to be able to test pr before merging so you shouldn't split room into 2 different request, same with koin and ui elements I can't review and debug them them properly since they not done yet